editor/scene_tree_dock.cpp

```diff
@@ -3415,7 +3415,7 @@ SceneTreeDock::SceneTreeDock(EditorNode *p_editor, Node *p_scene_root, EditorSel
 	menu_properties->connect("id_pressed", callable_mp(this, &SceneTreeDock::_property_selected));
 
 	clear_inherit_confirm = memnew(ConfirmationDialog);
-	clear_inherit_confirm->set_text(TTR("Clear Inheritance? (No Undo!)"));
+	clear_inherit_confirm->set_text(TTR("Clear Inheritance? (NO UNDO)"));
```

Comment:
```suggestion
	clear_inherit_confirm->set_text(TTR("Clear inheritance? (NO UNDO)"));
```
It's text and not title, so should use sentence case IMO.